src/2d/shallow/src1d.f90

```diff
@@ -9,8 +9,13 @@ subroutine src1d(meqn,mbc,mx1d,q1d,maux,aux1d,t,dt)
     use geoclaw_module, only: g => grav, coriolis_forcing, coriolis
     use geoclaw_module, only: friction_forcing, friction_depth
     use geoclaw_module, only: omega, coordinate_system, manning_coefficient
-    use geoclaw_module, only: manning_break, num_manning
-                              
+    use geoclaw_module, only: manning_break, num_manning, dry_tolerance
```

Comment:
variable friction coefficient in `setaux` is supported in `src2.f90` but not in `src1d.f90`?